yaml/roads.yaml

```diff
@@ -32,6 +31,14 @@ global:
       sport: {col: sport}
       man_made: {col: man_made}
       walking_network: {expr: "mz_hiking_network(osm_id)"}
+      bicycle_network: {expr: "mz_cycling_network(tags, osm_id)"}
+      oneway: {col: oneway}
```

Comment:
Let's ignore this for now (leave the `oneway:bicycle=no`) and pick it back up next release in: https://github.com/mapzen/vector-datasource/issues/782.